src/Content/Text/BBCode.php

```diff
@@ -1057,7 +1057,7 @@ private static function convertShareCallback(array $attributes, array $author_co
 		switch ($simplehtml) {
 			case self::API:
 				$text = ($is_quote_share? '<br>' : '') .
-				'<p><b><a href="' . $attributes['link'] . '">' . html_entity_decode('&#x2672; ', ENT_QUOTES, 'UTF-8') . ' ' . $author_contact['addr'] . "</a>:</b> </p>\n" .
+				'<b><a href="' . $attributes['link'] . '">' . html_entity_decode('&#x2672; ', ENT_QUOTES, 'UTF-8') . ' ' . $author_contact['addr'] . "</a>:</b><br>\n" .
```

Comment:
Please don't use `<br>` for styling. If you need the `<p>` bottom margin to be smaller, please add a CSS rule instead.

Reply:
This is specifically an output for the API. I tested several apps. Many are unable handling with `<p>` not to mention any CSS handling.

Reply:
Ugh.